force-app/main/default/classes/ContactTriggerHandler.cls

```diff
@@ -0,0 +1,11 @@
+public class ContactTriggerHandler {
```

Comment:
This is where all the work that was done in the trigger should be done. (if,else logic, for loops)